build/version.go

```diff
@@ -1,20 +1,24 @@
 package build
 
-import "os"
+import (
+	"os"
+
+	"github.com/filecoin-project/lotus/build/buildconstants"
+)
 
 type BuildVersion string
 
 var CurrentCommit string
-var BuildType int
+var BuildType = buildconstants.BuildType // Deprecated: Use buildconstants.BuildType instead
 
 const (
-	BuildDefault      = 0
-	BuildMainnet      = 0x1
-	Build2k           = 0x2
-	BuildDebug        = 0x3
-	BuildCalibnet     = 0x4
-	BuildInteropnet   = 0x5
-	BuildButterflynet = 0x7
+	BuildDefault      = buildconstants.BuildDefault      // Deprecated: Use buildconstants.BuildDefault instead
```

Comment:
What breaks if we just delete instead of leaving these pointer values around?

Reply:
nothing. I cleaned it up.